### PR TITLE
✨ Merging participant and demographic

### PR DESCRIPTION
**Pull request by parimalak**
- [x] Remove demographic entity
- [x] Add demographics fields in the participant entity
- [x] Update tests, migrations, data generator script and erd diagram

### CODE REVIEW

File: dataservice/api/__init__.py [hunk 1]
```diff
@@ -12,8 +12,8 @@
 from dataservice.api.participant import ParticipantListAPI
 from dataservice.api.family_relationship import FamilyRelationshipAPI
 from dataservice.api.family_relationship import FamilyRelationshipListAPI
-from dataservice.api.demographic import DemographicAPI
-from dataservice.api.demographic import DemographicListAPI
+# from dataservice.api.demographic import DemographicAPI
+# from dataservice.api.demographic import DemographicListAPI
```
Comment (dankolbman) [inline dataservice/api/__init__.py]: Remove these?

Comment (parimalak) [reply dataservice/api/__init__.py]: 👍

**Review by dankolbman — APPROVED** [review]
lgtm

**Review by znatty22 — APPROVED** [review]
Looks good! 👍

**Pull request merged**